config/rbac/role.yaml

```diff
@@ -18,6 +56,17 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - addons.cluster.x-k8s.io
+  resources:
+  - clusterresourcesets/status
```

Comment:
Are we actually reading/writing the status?
I believe we just write to the main spec, we don't do anything with the status?